src/main/java/us/kbase/mobu/ModuleBuilder.java

```diff
@@ -27,669 +27,704 @@
-			ModuleBuilder.showError("Init Error", "A module name is required.");
-			return 1;
-		}
-		String moduleName = StringUtils.join(initArgs.moduleNames, "_");
-
-		// Get username if available
-		String userName = null;
-		if (initArgs.userName != null)
-			userName = initArgs.userName;
+	@Command(name = "init", description = "Initialize a module in the current directory.")
+	public static class InitCommand extends Verbose implements Callable<Integer> {
+		
+		@Option(
+				paramLabel = "<user_name>",
+				names = {"-u", "--user"},
+				required = true,
+				description = "Provide a username to serve as the owner of this module."
+		)
+		String userName;
 
-		// Get chosen language
-		String language = ModuleInitializer.DEFAULT_LANGUAGE;
-		if (initArgs.language != null) {
-			language = initArgs.language;
-		}
-		try {
-			ModuleInitializer initer = new ModuleInitializer(moduleName, userName, language, initArgs.verbose);
-			initer.initialize(initArgs.example);
-		}
-		catch (Exception e) {
-			if (initArgs.verbose)
-				e.printStackTrace();
-			showError("Error while initializing module", e.getMessage());
-			return 1;
+		@Option(
+				names = {"-e", "--example"},
+				description = """
+						Include a fully featured example in your module. \
+						This generates an example set of code and configurations.\
+						""",
+				defaultValue = "false"
+		)
+		boolean example;
+		
+		@Option(
+				names = {"-l", "--language"},
+				description =
+						"The language for the module. Valid values: ${COMPLETION-CANDIDATES}.",
+				defaultValue = "python",
+				showDefaultValue = CommandLine.Help.Visibility.ALWAYS
+		)
+		Language language;
+
+		@Parameters(
+				paramLabel = "<module_name>",
+				description = "The name of the module to create.",
+				arity = "1"
+		)
+		String moduleName;
+
+		@Override
+		public Integer call() {
+			try {
+				final ModuleInitializer initer = new ModuleInitializer(
+						moduleName, userName, "" + language, verbose
+				);
+				initer.initialize(example);
+			}
+			catch (Exception e) {
+				showError("Error while initializing module", e.getMessage());
+				if (verbose) {
+					e.printStackTrace();
+				}
+				return 1;
+			}
+			return 0;
 		}
-		return 0;
+	}
+	
+	public static class ClientVersionsGroup {
+		@Option(
+				names = {"--clasyncver"},
+				description = """
+						Set the verion of code to run when making asynchronous calls via a \
+						client. Valid valus are a git commit hash or "dev", "beta", or "release". \
+						The targeted version must be registered in the KBase Catalog.\
+						"""
+		)
+		String clAsyncVer;
+
+		@Option(
+				names = {"--dynservver"},
+				description = """
+						Set clients to be built for use with KBase dynamic services \
+						(e.g. with URL lookup via the Service Wizard) with the specified \
+						dynamic service version. \
+						Valid valus are a git commit hash or "dev", "beta", or "release". \
```

Comment:
```
s/valus/values/g
```
(this appears several times)

Reply:
Fixed locally, will show up in a later PR